mrbgems/mruby-os-memsize/src/memsize.c

```diff
@@ -44,6 +44,22 @@ os_memsize_of_method(mrb_state* mrb, mrb_value method_obj)
   return size;
 }
 
+static mrb_bool
+obj_is_kind_of_checked(mrb_state *mrb, mrb_value obj, mrb_sym classname)
+{
+  mrb_value objclass = mrb_obj_value(mrb->object_class);
+
+  if (mrb_const_defined(mrb, objclass, classname)) {
+    mrb_value klass = mrb_const_get(mrb, objclass, classname);
```

Comment:
Probably you can use `mrb_class_get` here.

Reply:
I made the `obj_is_kind_of_checked()` function take a string as an argument.